### PR TITLE
Personal Portfolio Project setup for mobile

### PR DESCRIPTION
**Pull request by davidobi**
In this project, I set up the mobile skeleton of my project, using flex and added linters

### CODE REVIEW

**Review by keroloussamy — COMMENTED** [review]
Hi @davidobi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have linter checks. Please add `.github` folder, inside it add `workflows` folder, inside it add [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml), as required in [linter setup](https://github.com/microverseinc/linters-config/tree/master/html-css#set-up-github-actions)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                        <li><img src="img/cellular.png" alt="cellular"></li>
+                        <li><img src="img/wifi.png" alt="wifi"></li>
+                        <li><img src="img/capacity.png" alt="capacity"></li>
+                    </ul>
+                </nav>
+            </header>
+
+            <div class="logo">
+
+                <h2 class="my-logo">My Logo</h2>
```
Comment (micheaol) [inline index.html]: Please ensure that the logo has `a link (a tag)` according to the requirement.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+                <img class="ham" src="img/hamburger.png" alt="hamburger">
```
Comment (micheaol) [inline index.html]: Please ensure that the hamburger button has a `nav` tag according to the requirement.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                    <h2>I’m Mariane <br> Glad to see <br> you!</h2>
+
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some
+                        of my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate to contact me. </p>
+
+                </div>
+                <div class="connect">
+                    <h2>LET'S CONNECT</h2>
+                </div>
```
Comment (micheaol) [inline index.html]: Please ensure that this section is wrapped in a `section` tag according to the requirement and the content is aligned with flexbox.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                <h2 class="my-logo">My Logo</h2>
+
+                <img class="ham" src="img/hamburger.png" alt="hamburger">
+
+            </div>
+
+            <div class="description">
+
+                <div class="name">
+                    <h2>I’m Mariane <br> Glad to see <br> you!</h2>
```
Comment (micheaol) [inline index.html]: Please ensure that the main title (e.g., "I'm Marianne ...") is a `h1` tag.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        <section class="pg-1">
+            <header>
+                <img class="img" src="img/time.png" alt="time">
+
+                <nav>
+                    <ul>
+                        <li><img src="img/cellular.png" alt="cellular"></li>
+                        <li><img src="img/wifi.png" alt="wifi"></li>
+                        <li><img src="img/capacity.png" alt="capacity"></li>
+                    </ul>
```
Comment (micheaol) [inline index.html]: Please remove the `time, wifi, cellular and capacity` they are not part of the design.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @davidobi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️

- _Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 4 above)
Comment (davidobi) [reply index.html]: @micheaol  Thanks for your suggestion. Please the requirements suggest h2 ta
Screenshot 2022-02-10 at 09 33 55
g

**Comment by davidobi** [comment]
@micheaol. Thanks for your review. All requested changes have been successfully implemented. Please check

File: index.html (hunk 4 above)
Comment (micheaol) [reply index.html]: @davidobi ,
Kindly make the requested changes it's according to the requirement.

Comment (davidobi) [reply index.html:32]: @micheaol am sorry it doesn't give me the right styling when I used H1 and if you can share a copy of the requirements with me, I would be glad to go through thanks

Comment (micheaol) [reply index.html]: Please find below as requested.
image

Comment (davidobi) [reply index.html]: @micheaol 
Thanks for the image. I appreciate and am sorry for the stress we didn't have the requirements to work with and was asked to work with the Figma diagram.

it has been implemented accordingly please review.

**Comment by davidobi** [comment]
@micheaol Please can I merge the project?

**Comment by micheaol** [comment]
@davidobi,
Kindly resubmit for re-review once all changes are attended to.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+<body>
+    <section class="container">
+        <section class="pg-1">
+            <header>
+                
+            </header>
+
+            <div class="logo">
+
+                <h2 class="my-logo"><a href="#" target="_blank" rel="noopener noreferrer">My Logo</a></h2>
```
Comment (mateo951) [inline index.html:21]: Addressed ✔

File: index.html [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+                <nav>
+                    <img class="ham" src="img/hamburger.png" alt="hamburger">
+                </nav>
```
Comment (mateo951) [inline index.html:24]: Addressed ✔

File: index.html [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+                
+
+            </div>
+
+            <section class="description">
+
+                <div class="name">
+                    <h1>I’m Mariane <br> Glad to see <br> you!</h1>
```
Comment (mateo951) [inline index.html:32]: Addressed 😎

**Review by mateo951 — COMMENTED** [review]
### Hi @davidobi, 

#### Please don't merge your PR's before getting approved next time 😉

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (davidobi) [reply index.html:21]: @mateo951 Sorry about that. I would definitely wait for approval henceforth. Thanks so much.

**Pull request merged**